blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -139,7 +134,7 @@ export class Step extends Component {
             liveFormat={t('common.date.duration.format', { defaultValue: 'm[ minutes] s[ seconds]' })}
             hintFormat={t('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' })}
         />);
-        return (<div className={logConsoleClass} key={this.pager.currentLogUrl}>
+        return (<div className={logConsoleClass}>
```

Comment:
After making the above change, I noticed that when the log data loaded, the slide/open animation would play again. It wasn't terrible but it looked a little funny. I noticed that the value of `this.pager.currentLogUrl` was undefined before the data fetch, and then populated after. This caused the component to be created/redestroyed, so the `ResultItem` was playing its animation again. This fix seems appropriate but I am curious if there is a specific reason you set a `key` on this component. It seems unnecessary since the parent `Steps` component is already defining a `key` on each Step?